editor/editor_help_search.cpp

```diff
@@ -401,8 +401,46 @@ bool EditorHelpSearch::Runner::_phase_match_classes() {
 					}
 				}
 			}
-			matches[class_doc.name] = match;
 		}
+		// Otherwise if term is empty add all requested members
```

Comment:
We're trying to standardize comment style around having them also end sentences with a dot.

Reply:
Thank you! Will do